walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,7 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    name = name.replaceAll("[^a-zA-Z0-9]", "");
```

Comment:
Nice use of a regex here - you've trimmed the whitespace and removed characters such as: @#$% in one fell swoop!

Reply:
thanks!